nipype/algorithms/confounds.py

```diff
@@ -654,10 +654,10 @@ def _run_interface(self, runtime):
         else:
             components_criterion = 6
             IFLOGGER.warning(
-                "`num_components` and `variance_threshold` are "
+                "``num_components`` and ``variance_threshold`` are "
                 "not defined. Setting number of components to 6 "
                 "for backward compatibility. Please set either "
-                "`num_components` or `variance_threshold`, as "
+                "``num_components`` or ``variance_threshold``, as "
```

Comment:
These aren't RST.

Reply:
Ups! Search and replace... Would you roll them back?